README.md

```diff
@@ -277,6 +277,10 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Publishing code coverage to CodeClimate](https://github.com/paambaati/codeclimate-action)
 - [Update repository go report card](https://github.com/creekorful/goreportcard-action)
 
+#### Code Quality
```

Comment:
The goimports command changes code, as opposed to just checking code statically. It should be moved to a different section.

Reply:
@jessicalostinspace I used title casing and improve the documentation a bit

Reply:
Thank you for doing that